layouts/_default/author.html

```diff
@@ -257,7 +257,10 @@ <h3 class="my-2">
       {{ partialCached "tw/modal-sticky.html" . }}
     {{ end }}
     {{ if .Param "newsletter-active" }}
-      {{ partialCached "tw/modal-newsletter.html" . }}
+      {{ partialCached "tw/modal-ribbon.html" ( dict
```

Comment:
If you use partialCached instead of plain partial, does it ever get confused about which modal is which?

Reply:
Good call. We're now just loading partial for the modal ribbon. https://github.com/spotlightpa/poor-richard/pull/367/commits/7733d03fd4527a3d242088c275721750ae4d8e0c